dataactcore/scripts/ad_hoc/load_full_sam_subaward_dump.py

```diff
@@ -0,0 +1,234 @@
+
+    if CONFIG_BROKER['use_aws']:
+        s3_client = boto3.client('s3', region_name=CONFIG_BROKER['aws_region'])
+        subaward_file = s3_client.generate_presigned_url(ClientMethod='get_object',
+                                                         Params={'Bucket': CONFIG_BROKER['data_extracts_bucket'],
+                                                                 'Key': f'sam_subaward_bulk_dump/{filename}'},
+                                                         ExpiresIn=600)
+    else:
+        subaward_file = os.path.join(CONFIG_BROKER['path'], 'dataactvalidator', 'config', filename)
+
+    data = pd.read_csv(subaward_file)
+    data = clean_data(
+        data,
+        file_filters[file_type]['model'],
+        {
+            'subawarddescription': 'description',
+            'subawardreportid': 'subaward_report_id',
+            'subawardreportnumber': 'subaward_report_number',
+            'submitteddate': 'date_submitted',
+            'subawardnumber': 'award_number',
+            'subawardamount': 'award_amount',
+            'subawarddate': 'action_date',
+            'subparentuei': 'parent_uei',
+
+            'vendorphysicaladdress_streetaddress': 'legal_entity_address_line1',
+            'vendorphysicaladdress_streetaddress2': 'legal_entity_address_line2',
+            'vendor_physicaladdress_city': 'legal_entity_city_name',
+            'vendor_physicaladdress_congressionaldistrict': 'legal_entity_congressional',
+            'vendor_physicaladdress_state_code': 'legal_entity_state_code',
+            'vendor_physicaladdress_state_name': 'legal_entity_state_name',
+            'vendor_physicaladdress_country_code': 'legal_entity_country_code',
+            'vendor_physicaladdress_country_name': 'legal_entity_country_name',
+            'vendor_physicaladdress_zip': 'legal_entity_zip_code',
+
+            'sub_place_of_performance_streetaddress': 'ppop_address_line1',
+            'sub_place_of_performance_city': 'ppop_city_name',
+            'sub_place_of_performance_congressional_district': 'ppop_congressional_district',
+            'sub_place_of_performance_state_code': 'ppop_state_code',
+            'sub_place_of_performance_state_name': 'ppop_state_name',
+            'sub_place_of_performance_country_code': 'ppop_country_code',
+            'sub_place_of_performance_country_name': 'ppop_country_name',
+            'sub_place_of_performance_zip': 'ppop_zip_code'
+        } | file_filters[file_type]['mapping'],
+        {}
+    )
+
+    data['business_types_codes'] = data.apply(lambda row: string_to_array(row, 'business_types_codes'), axis=1)
+    #data['business_types_names'] = data.apply(lambda row: string_to_array(row, 'business_types_names'), axis=1)
+    # Clear any lingering np.nan's
+    data = data.replace({np.nan: None})
+
+    # TODO: Remove this once we receive these from the file and run the string_to_array function on it instead
```

Comment:
Does this need to update given they confirmed it will be included in the full dump on monday?